### DISCUSSION TITLE
Should we use SvelteKit in our Project?

### DISCUSSION THREAD

**Discussion by NathanGrenier** [NathanGrenier/SOEN-341#31] [Polls]
After completing the [SvelteKit learning path](https://learn.svelte.dev/tutorial), vote on whether we should use it in our project.

If you're short on time, I recommend completing parts 1 and 3 of the [learning path](https://learn.svelte.dev/tutorial) to get a quick but sufficient understanding of the framework.

**Comment by briantkatch** [comment]
No preference
